packages/sdk/schema/src/view/view.ts

```diff
@@ -68,10 +69,19 @@ const View_ = Schema.Struct({
 
   /**
    * Query used to retrieve data.
-   * This includes the base type that the view schema (above) references.
```

Comment:
Are we adding the _ suffix as a standard?

Reply:
not sure what this is in reference to